src/cards/send_mana.ts

```diff
@@ -21,10 +21,10 @@ const spell: Spell = {
     category: CardCategory.Mana,
     sfx: 'potionPickupMana',
     supportQuantity: true,
-    manaCost: 30,
+    manaCost: 20,
```

Comment:
This should be 30.  We talked about this at length, why the change?  It's super broken (OP with timemason) if the send to receive ratio is 1:1

Reply:
I know, but I do still really think the the expense scaling fixes that.

I gave the 30 base cost a good try and I just don't think it's the play. It feels too expensive to use in those clutch moments we talked about, and it feels too expensive/niche to ever choose over other spells when picking upgrades.